packages/common/src/directives/ng_optimized_image/lcp_image_observer.ts

```diff
@@ -42,14 +45,13 @@ export class LCPImageObserver implements OnDestroy {
   // Map of full image URLs -> original `ngSrc` values.
   private images = new Map<string, ObservedImageState>();
 
-  private window: Window | null = null;
   private observer: PerformanceObserver | null = null;
+  private platformLocation = inject(PlatformLocation);
```

Comment:
You can also change this to the below to save some extra bytes.

```suggestion
  private baseUrl = inject(PlatformLocation).href;
```